tests/dummy/app/templates/linkedin/share.hbs

```diff
@@ -24,6 +25,14 @@
   {{linkedin-share url='http://example.com/share-things'}}
 </div>
 
+<h2>Display Share count</h2>
+
+<p>Count can be displayed either on top or on the right of the button.</p>
+
+<code>
+  \{{linkedin-share count="top"}}<br>
+</code>
+
```

Comment:
Can you add a visual example in addition to the code example? Thanks @Peter-Barrett !